app/components/pan-zoom.jsx

```diff
@@ -45,115 +35,41 @@ const PanZoom = React.createClass({
       this.root.addEventListener('keydown', this.frameKeyPan);
       this.root.addEventListener('wheel', this.wheelZoom);
     }
-  },
+    this.zoomReset();
+  }
 
-  componentWillUpdate(newProps) {
-    if (newProps.frameDimensions === this.props.frameDimensions) return;
-    this.setState({
-      viewBoxDimensions: {
-        width: newProps.frameDimensions.width,
-        height: newProps.frameDimensions.height,
-        x: 0,
-        y: 0
-      }
-    });
-  },
+  componentDidUpdate(oldProps) {
```

Comment:
`zoomReset` triggers an update and render, so `componentWillUpdate` might be safer here to avoid recursive loops.